packages/data-stores/tsconfig.json

```diff
@@ -31,6 +31,6 @@
 		"noEmitHelpers": true,
 		"importHelpers": true
 	},
-	"files": [ "./src/index.ts" ],
+	"files": [ "./src/index.ts", "./global.d.ts" ],
```

Comment:
🤔 We could add a `@types` directory in the Calypso root to share type declarations across the monorepo. I think if we add the directory to `typeRoots` and name the file `wpcom-proxy-request.d.ts`, it would pull types from there.

Reply:
Having shared types in the monorepo would be great. If `wpcom-proxy-request` is coming to the monorepo though (pbxlJb-6v-p2#comment-51) I'd almost just wait until then and add the `*.d.ts` to it's directory.

Reply:
Agreed that with the imminent ingression, this one type isn't valuable. However, I'm quite interested in discovering pain points and figuring out these patterns that empower development across the monorepo. It's likely we'll come across untyped packages again.

No need to sort it _all_ out in this PR, of course 🙂